src/main/java/org/alliancegenome/curation_api/model/entities/DiseaseAnnotation.java

```diff
@@ -70,7 +70,7 @@
     @JsonView({View.FieldsAndLists.class})
     private List<EcoTerm> evidenceCodes;
     
-    @IndexedEmbedded(includeDepth = 1)
+    @IndexedEmbedded(includeDepth = 2)
```

Comment:
@adamgibs did you test this change locally?

Reply:
Yep, it seemed to work fine.